parse_node/parse_import_declaration.ts

```diff
@@ -102,92 +101,124 @@ export const parseImportDeclaration = (
 
-    for (const element of bindings.elements) {
-      const type = props.program.getTypeChecker().getTypeAtLocation(element)
+    // TODO rewrite this using new project obj
 
-      // TODO rewrite this using new project obj
+    if (isEnumType(type)) {
+      const { resPath, enumName } = getImportResPathForEnum(type, props)
 
-      if (isEnumType(type)) {
-        const { resPath, enumName } = getImportResPathForEnum(type, props)
+      imports.push({ importedName: enumName, resPath: resPath, type: "enum" })
+    } else if (type.symbol?.name === "PackedScene") {
+      const importedName = element.name.text
+      const className = importedName.slice(0, -"Tscn".length)
+      const resPath = props.project
+        .godotScenes()
+        .find((scene) => scene.name === className)?.resPath
 
-        imports.push({ importedName: enumName, resPath: resPath, type: "enum" })
-      } else if (type.symbol?.name === "PackedScene") {
-        const importedName = element.name.text
-        const className = importedName.slice(0, -"Tscn".length)
-        const resPath = props.project
-          .godotScenes()
-          .find((scene) => scene.name === className)?.resPath
+      if (!resPath) {
+        continue
+      }
 
-        if (!resPath) {
+      imports.push({
+        importedName: importedName,
+        resPath: resPath,
+        type: "scene",
+      })
+    } else {
+      const importedSourceFile = props.project
+        .sourceFiles()
+        .find((sf) => sf.fsPath === pathToImportedTs)
+
+      if (!importedSourceFile) {
+        if (pathToImportedTs.includes("@")) {
           continue
         }
 
-        imports.push({
-          importedName: importedName,
-          resPath: resPath,
-          type: "scene",
+        addError({
+          error: ErrorName.InvalidNumber,
+          location: node,
+          description: `Import ${pathToImportedTs} not found.`,
+          stack: new Error().stack ?? "",
         })
-      } else {
-        const importedSourceFile = props.project
-          .sourceFiles()
-          .find((sf) => sf.fsPath === pathToImportedTs)
-
-        if (!importedSourceFile) {
-          if (pathToImportedTs.includes("@")) {
-            continue
-          }
 
-          addError({
-            error: ErrorName.InvalidNumber,
-            location: node,
-            description: `Import ${pathToImportedTs} not found.`,
-            stack: new Error().stack ?? "",
-          })
+        continue
+      }
 
-          continue
-        }
+      let typeString = props.program.getTypeChecker().typeToString(type)
 
-        let typeString = props.program.getTypeChecker().typeToString(type)
+      if (typeString.startsWith("typeof ")) {
+        typeString = typeString.slice("typeof ".length)
+      }
 
-        if (typeString.startsWith("typeof ")) {
-          typeString = typeString.slice("typeof ".length)
-        }
+      if (typeString === "(Missing)") {
```

Comment:
This showed up when TypeScript could not provide a type because I had errors in my game project code. This was very specific case and I do not remember right now what caused it, but it can happen according to this TypeScript code: 
* https://github.com/microsoft/TypeScript/blob/1bc124e6359d8e7fa54206c8b58d324bd8b80beb/src/compiler/utilities.ts#L960-L965
* https://github.com/microsoft/TypeScript/blob/1bc124e6359d8e7fa54206c8b58d324bd8b80beb/src/compiler/transformers/declarations.ts#L175-L180)